featurebyte/query_graph/sql/adapter/bigquery.py

```diff
@@ -332,7 +332,7 @@ def radian_expr(cls, expr: Expression) -> Expression:
     @classmethod
     def to_timestamp_from_string(cls, expr: Expression, format_string: str) -> Expression:
         return expressions.Anonymous(
-            this="PARSE_DATETIME",
+            this="PARSE_TIMESTAMP",
             expressions=[make_literal_value(format_string), expr],
         )
```

Comment:
@YS-L I change this to `PARSE_TIMESTAMP` as `PARSE_DATETIME` will throw error if the format string contains timezone substring.